src/sys.c

```diff
@@ -652,6 +652,29 @@ JL_DLLEXPORT int jl_cpu_threads(void) JL_NOTSAFEPOINT
 #endif
 }
 
+int jl_effective_threads(void) JL_NOTSAFEPOINT
+{
+    int cpu = jl_cpu_threads();
+    int masksize = uv_cpumask_size();
+    if (masksize < 0)
```

Comment:
```suggestion
    if (masksize < 0 || jl_running_under_rr(0))
```

Reply:
Since
```
$ /data/vtjnash/rr/bin/rr ./julia test/print_process_affinity.jl 
rr: Saving execution to trace directory `/home/vtjnash/.local/share/rr/julia-399'.
20
```
vs
```
$ /data/vtjnash/rr/bin/rr record -u ./julia test/print_process_affinity.jl 
rr: Saving execution to trace directory `/home/vtjnash/.local/share/rr/julia-400'.
1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24
```